config/locales/diaspora/en.yml

```diff
@@ -427,9 +427,9 @@ en:
       restrict_posts_i_see_q: "Can I restrict the posts in my stream to just those from certain aspects?"
       restrict_posts_i_see_a: "Yes. Click on “My aspects” in the side-bar and then click individual aspects in the list to select or deselect them. Only the posts by people in the selected aspects will appear in your stream."
       rename_aspect_q: "Can I rename an aspect?"
-      rename_aspect_a: "Yes. In your list of aspects on the left side of the main page, point your mouse at the aspect you want to rename. Click the little “Edit” pencil that appears to the right. You will be taken to the page for this aspect. Click the edit pencil next to the aspect name at the top of this page, the change the name and press “Update”."
+      rename_aspect_a: "Yes. Go to the “Contacts” page form the user menu and select the aspect you want to rename. You can also go there from the stream : click on “My aspects” on the left side of the main page, point your mouse at the aspect you want to rename and click the little “Edit” pencil that appears to the right. You will be taken to the page for this aspect. Click the edit pencil next to the aspect name at the top of this page, then change the name and press “Update”."
       delete_aspect_q: "How do I delete an aspect?"
-      delete_aspect_a: "In your list of aspects on the left side of the main page, point your mouse at the aspect you want to delete. Click the little “Edit” pencil that appears on the right. Click the “Delete this aspect” button in the box that appears."
+      delete_aspect_a: "Like to rename an aspect, go to the “Contacts” page and select the aspect either by click on “Contacts” in the user menu or on the pencil appearing at the right of the aspect in your list of aspects on the left side of the main page. Once on the contact page with the aspect selected, click on the trash icon on the top right of the page."
```

Comment:
I don't think we should say 'like another answer', because that means the user would have to find and read that answer before understanding what to do for this answer. The help section is designed to be clicked and read individually rather than as a single document.